AI-and-Analytics/Features-and-Functionality/IntelTensorFlow_InferenceOptimization/sample.json

```diff
@@ -0,0 +1,20 @@
+{
+	"guid": "9d32f194-8667-41d3-865d-d43e9983c471",
+	"name": "Tutorial : Optimize TensorFlow pre-trained model for inference",
+	"categories": ["Toolkit/oneAPI AI And Analytics/Features And Functionality"],
+	"description": "This tutorial will guide you how to optimize a pre-trained model for a better inference performance, and also analyze the model pb files before and after the inference optimizations.",
+	"builder": ["cli"],
+	"languages": [{
+		"python": {}
```

Comment:
As this is python, I will need a bandit scan from you prior to merging

Reply:
**just got below warning. 
any suggestion for possible fix or replacement of os.system?**

Run started:2021-05-17 23:03:03.590150

Test results:
>> Issue: [B605:start_process_with_a_shell] Starting a process with a shell, possible injection detected, security issue.
   Severity: High   Confidence: High
   Location: IntelTensorFlow_InferenceOptimization/scripts/profile_utils.py:21
20      def do_command(command):
21          os.system(command)
22

--------------------------------------------------

Code scanned:
        Total lines of code: 1421
        Total lines skipped (#nosec): 0

Run metrics:
        Total issues (by severity):
                Undefined: 0
                Low: 0
                Medium: 0
                High: 1
        Total issues (by confidence):
                Undefined: 0
                Low: 0
                Medium: 0
                High: 1
Files skipped (0):

Reply:
take a look at: https://docs.openstack.org/bandit/1.4.0/plugins/start_process_with_a_shell.html

Reply:
Remove os.system for bandit

new result : 
ality$ bandit IntelTensorFlow_InferenceOptimization/
[main]  INFO    profile include tests: None
[main]  INFO    profile exclude tests: None
[main]  INFO    cli include tests: None
[main]  INFO    cli exclude tests: None
[main]  INFO    running on Python 2.7.17
[manager]       WARNING Skipping directory (IntelTensorFlow_InferenceOptimization/), use -r flag to scan contents
Run started:2021-05-19 23:54:31.366297

Test results:
        No issues identified.

Code scanned:
        Total lines of code: 0
        Total lines skipped (#nosec): 0

Run metrics:
        Total issues (by severity):
                Undefined: 0
                Low: 0
                Medium: 0
                High: 0
        Total issues (by confidence):
                Undefined: 0
                Low: 0
                Medium: 0
                High: 0
Files skipped (0):